app/src/main/java/com/sincalexandrudaniel/newsreader/NewsReaderActivity.java

```diff
@@ -0,0 +1,27 @@
+package com.sincalexandrudaniel.newsreader;
+
+import android.os.Bundle;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.navigation.NavController;
+import androidx.navigation.fragment.NavHostFragment;
+
+public class NewsReaderActivity extends AppCompatActivity {
+    private NavHostFragment navHostFragment;
+    private NavController navController;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.main_activity);
+//        if (savedInstanceState == null) {
```

Comment:
let's cleanup this code, we don't need commented code, we got git for checking out the code history